src/lib/slider/slider.ts

```diff
@@ -182,6 +182,7 @@ export class MdSlider implements AfterContentInit, ControlValueAccessor {
 
     this.isActive = true;
     this.isSliding = false;
+    this._sliderDimensions = this._renderer.getSliderDimensions();
```

Comment:
If this is only necessary in one handler, I would just leave it in the click handler. Also needs a comment explaining why the dimensions need to be recalculated here.